### PR TITLE
remove endowed positions sub page dropdown

### PR DESCRIPTION
**Pull request by wmui51**
# READY FOR REVIEW

# Summary
- Request to remove the dropdown in the Endowed Positions school/center pages.
- Styled the search to be consistent with the Endowed Positions Landing page search (currently on Dev).

# Review By September 23, 2022
- We're hoping to push this update, along with the update to add the search to the Landing page (already on Dev) to Production by the end of the month.

# Criticality - 4
- This update impacts the Endowed Positions sub-pages and provides visual continuity with the search interface on the Landing page.

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to /endowed-positions/graduate-school-of-business (or any other school/center page)
2. Check to see that the dropdown navigation is no longer there and only a search input is still there.
3. Test search input to see if it works.
4. Check search results page with same steps as step 2.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by forrestglick** [comment]
@wmui51 Thanks for making the updates. I reviewed the center-aligned copy/search with Donor Relations, and we agree it would be best to left-align the block. I hope it's not too much more work to do so. Thanks!

**Comment by forrestglick** [comment]
@wmui51 Donor Relations has reviewed and approved. We are good to go.

File: src/scss/components/endowed-positions/index.scss [hunk 1]
```diff
@@ -76,38 +79,23 @@
...
-    padding: 0 38px 0 18px;
-    width: calc(100% - 55px);
-  }
-
   &__input {
     box-sizing: border-box;
     border: 1px solid #D2D3D4;
     border-radius: 3px;
+    flex-grow: 0;
     font-size: 18px;
```
Comment (yvonnetangsu) [inline src/scss/components/endowed-positions/index.scss:87]:
```suggestion
    font-size: 1.8rem;
```

**Review by yvonnetangsu — APPROVED** [review]
This works as described! If you could please click the "commit suggestion" button for the font size unit change that I suggested above, then this is GTG 😄 We recommend sticking to rem (1rem = 10px on this site) unit especially for font sizes because of accessibility.

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu i will update the font size, thanks for looking.

If you click the "Commit suggestion" button above, it will merge that right in 😄

**Comment by yvonnetangsu** [comment]
> oh wow, did not know that was a feature. done and thanks again!

Yeah it's so convenient isn't it? 😄 And you're welcome, @wmui51 !

**Pull request merged**